### PR TITLE
BraveSoftware.Brave version 1.38.107

### PR DESCRIPTION
**Pull request by gr4vem1nd**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/60023)

### CODE REVIEW

File: manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.Brave
+PackageVersion: 1.38.107
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://github.com/brave/brave-browser/releases/download/v1.38.107/BraveBrowserStandaloneSetup.exe
+  InstallerSha256: 541640B27DCA3398973B48717D9C481A84F6C559E4A7C598BE6BAF28E2A0751F
```
Comment (jedieaston) [inline manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.installer.yaml:10]:
```suggestion
  InstallerUrl: https://github.com/brave/brave-browser/releases/download/v1.38.107/BraveBrowserStandaloneSilentSetup.exe
  InstallerSha256: F4C099B69658659A57F62FD6C3DD339C4DED6660E0747EDBBA8F0DA13982B336
```

File: manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  - http
+  - https
+  FileExtensions:
+  - htm
+  - html
+  - url
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://github.com/brave/brave-browser/releases/download/v1.38.107/BraveBrowserStandaloneSetup32.exe
+  InstallerSha256: 5715ECB9D1F2C7882A36E0EE77E80226B55916980E636AB2DA1A2A925AA17D83
```
Comment (jedieaston) [inline manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.installer.yaml:24]:
```suggestion
  InstallerUrl: https://github.com/brave/brave-browser/releases/download/v1.38.107/BraveBrowserStandaloneSilentSetup32.exe
  InstallerSha256: E66D9448F4731E15C2FAE9107EE578F36CBFE6E42FCCA4E57AEE4EFFC939684C
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
They ship a separate silent installer :(

**Comment by gr4vem1nd** [comment]
There are also silent standalone setup files on the git release pages. Maybe they would have been better, but I didn't know how to specify both.

https://github.com/brave/brave-browser/releases/tag/v1.38.107

File: manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.Brave
+PackageVersion: 1.38.107
+PackageLocale: en-US
+Publisher: Brave Software Inc.
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.locale.en-US.yaml:7]:
```suggestion
Publisher: Brave Software Inc
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.Brave
+PackageVersion: 1.38.107
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.installer.yaml:5]:
```suggestion
PackageVersion: 101.1.38.107
```

File: manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.Brave
+PackageVersion: 1.38.107
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 101.1.38.107
```

File: manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.Brave
+PackageVersion: 1.38.107
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.yaml:5]:
```suggestion
PackageVersion: 101.1.38.107
```

Comment (ItzLevvie) [reply manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.yaml:5]: This must match Control Panel.

File: manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.locale.en-US.yaml (hunk 5 above)
Comment (ItzLevvie) [reply manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.locale.en-US.yaml:5]: This must match Control Panel.

File: manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.installer.yaml (hunk 4 above)
Comment (ItzLevvie) [reply manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.installer.yaml:5]: This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,37 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.Brave
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.installer.yaml:4]:
```suggestion
PackageIdentifier: BraveSoftware.BraveBrowser
```

File: manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.Brave
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: BraveSoftware.BraveBrowser
```

File: manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.yaml [hunk 9]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.Brave
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.yaml:4]:
```suggestion
PackageIdentifier: BraveSoftware.BraveBrowser
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.installer.yaml [hunk 10]
```diff
@@ -0,0 +1,37 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.Brave
+PackageVersion: 101.1.38.107
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/Brave/1.38.107/BraveSoftware.Brave.installer.yaml:5]:
```suggestion
PackageVersion: 101.1.38.107
ProductCode: BraveSoftware Brave-Browser
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You will also need to move the package to https://github.com/microsoft/winget-pkgs/tree/master/manifests/b/BraveSoftware/BraveBrowser and update the file names so you don't get the `Manifest-Validation-Error` label.

**Comment by gr4vem1nd** [comment]
> You will also need to move the package to https://github.com/microsoft/winget-pkgs/tree/master/manifests/b/BraveSoftware/BraveBrowser and update the file names so you don't get the `Manifest-Validation-Error` label.

Okay, thanks, I will change that!

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**